recipes-ni/nirtcfg-tests/nirtcfg-tests_1.0.bb

```diff
@@ -1,45 +0,0 @@
-SUMMARY = "Ptests for the nirtcfg utility"
-DESCRIPTION = "\
-Installs ptests for the nirtcfg utilty."
-
-SECTION = "tests"
-LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
-
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Lost this line.

Reply:
`./files` and `./${PN}` are in the `FILES` path by default for recipes. Whoever authored this recipe added them to `FILESEXTRAPATHS`, because that is what you need to do for **bbappend** files, but not for `.bb`s. It isn't needed.